package.json

```diff
@@ -94,5 +93,8 @@
     "re:build": "rescript",
     "re:watch": "rescript clean && rescript build -w",
     "wds": "TAILWIND_MODE=watch bin/webpack-dev-server"
+  },
+  "devDependencies": {
+    "@pnotify/mobile": "^5.2.0"
```

Comment:
Let's move this into `dependencies`. Our project doesn't need `devDependencies` since there's no differentiation between environments for client-side code - pretty much all dependencies are required for compilation of production code.